drivers/counter/counter_gecko_rtcc.c

```diff
@@ -91,10 +91,12 @@ static u32_t counter_gecko_read(struct device *dev)
 }
 
 static int counter_gecko_set_top_value(struct device *dev, u32_t ticks,
-				       counter_top_callback_t callback,
-				       void *user_data)
+				       const struct counter_top_cfg *top_cfg)
```

Comment:
Could we use `cfg` instead of `top_cfg` like in all other drivers. To make it more consistent.

Reply:
done